nailgun/entities.py

```diff
@@ -2666,6 +2666,22 @@ def disable_rhrepo(self, base_arch,
         )
         return _handle_response(response, self._server_config, synchronous)
 
+    def list_available_rhrepos(self, reposet_id):
```

Comment:
@abalakh Sure, let's use `repository_sets_available_repositories` instead. That works.